depends/packages/openssl.mk

```diff
@@ -0,0 +1,45 @@
+# Pretty much taken verbatim from Bitcoin Core
```

Comment:
I'll think about. I totally understand why the core devs rejected the upgrade. (Wladimir is right. The sooner OpenSSL is chucked, the better off everybody will be!) In our case, we don't use it in Armory, where Crypto++ does all the verification and such (although I suspect we'll migrate to libsecp256k1 once it's ready for primetime). OpenSSL would just be used for external connections, something Armory doesn't really care about right now but may in the future. That said, I'll give this some more thought.